common/temp_modules/npmx-gulp-core-build-karma/package.json

```diff
@@ -29,6 +29,6 @@
     "webpack": "~1.13.0"
   },
   "npmxDependencies": {
-    "@microsoft/gulp-core-build": ">=1.0.1 <2.0.0"

```

Comment:
Nope, they are supposed to be checked in, at least in the current model.. However, I'm not sure why we couldn't just generate these on the fly before we run install